### DISCUSSION TITLE
[Choix du statut] Code couleurs et signes des statuts disponibles

### DISCUSSION THREAD

**Discussion by glopezrios** [betagouv/mon-entreprise#2959] [General]
Remarque d'un usager, concernant [l'aide au choix du statut](https://mon-entreprise.urssaf.fr/assistants/choix-du-statut/recherche-activite): la signification du code couleur et les signes apposés aux statuts n'est pas expliquée.

Cela peut être compris de manière intuitif, mais il n'est pas précisé que chaque couleur correspond à une type d'entreprise. Par exemple, les sociétés libérales (SELARL, SELARLU, SELAS, SELASU) sont en vert. L'usager aurait aimé avoir un "tableau explicatif".

Le choix des petits sigles (rond, triangle, losange...) semble obéir à la même logique que les couleurs. Mais il pourrait par exemple suivre une autre logique, par exemple pour différencier les entreprises, les sociétés unipersonnelles et les sociétés avec plusieurs associés.

**Comment by logic-fabric** [comment]
> Le choix des petits sigles (rond, triangle, losange...) semble obéir à la même logique que les couleurs.

Ces petits symboles sont présents pour satisfaire un critère d'accessibilité, le critère 3.1 du RGAA : _"Dans chaque page web, l’information ne doit pas être donnée uniquement par la couleur."_
C'est un besoin pour les utilisateurs daltoniens (8% des hommes et 0,5% des femmes).

Mais la signification des combinaisons symbole/fond coloré mériterait une légende oui. 🙂

**Comment by logic-fabric** [comment]
**suggestion:** En lien avec cette issue, le tooltip qui apparaît au survol d'un sigle de statut n'est déclenchable que sur une trop petite zone, celle du sigle.

![image](https://github.com/betagouv/mon-entreprise/assets/69042083/b76c0fa0-40b5-487a-97a7-80a2aa674127)

C'est un autre souci d'accessibilité, pour les utilisateurs souffrant de tremblements par exemple.
Et ce serait plus confortable pour tous les utilisateurs d'avoir une plus large zone de déclenchement.

Le HTML de ce badge pourrait être repensé, pour que l'attribut `title` soit porté par le badge lui-même.
Le badge pourrait par exemple être un `<abbr>` dans son ensemble, contenant des `<span>` pour le symbole et le texte. 🙂

**Comment by glopezrios** [comment]
A prolonger dans #3013
